config/routes.rb

```diff
@@ -15,7 +15,7 @@
 
   get '/login', to: 'sessions#new'
 
-  post '/login', to: 'sessions#create', as: 'sessions'
+  post '/login', to: 'sessions#create'#, as: 'sessions'
```

Comment:
Missing space after #.<br>Put a space before an end-of-line comment.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.